stdlib/typing.pyi

```diff
@@ -114,9 +114,7 @@ FrozenSet = _Alias()
 Counter = _Alias()
 Deque = _Alias()
 ChainMap = _Alias()
-
-if sys.version_info >= (3, 7):
-    OrderedDict = _Alias()
+OrderedDict = _Alias()
```

Comment:
We can't change this, since Python 3.6's typing does not include `OrderedDict`.

Reply:
Ah, I got confused based on how the other ones were implemented. How should I add it to `typing_extensions` then? Is importing `_Alias()` from `typing` and using that acceptable?

Reply:
Either that or just copy `_Alias` to `typing_extensions` as well, whatever you prefer.